### PR TITLE
Upload banner for topic

### PR DESCRIPTION
**Pull request by fuermosi777**

### CODE REVIEW

File: actions/AudioActions.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
     uploadFile(context, payload, done) {
         let file = payload.file;
         const isNoteAudio = payload.isNoteAudio;
         if ( file.type === 'audio/mp3') {
-            const req = request.post('/upload');
-            req.attach(file.name, file);
-            req.end((err, res) => {
+            const req = request
+            .post('/upload')
+            .attach(file.name, file, 'whatever.mp3', {contentType: 'audio/mp3'})
```
Comment (jacgrady1) [inline actions/AudioActions.js]: whatever.mp3?

Comment (fuermosi777) [reply actions/AudioActions.js]: just a placeholder. it will be renamed anyway

Comment (jacgrady1) [reply actions/AudioActions.js]: okie then.

**Comment by jacgrady1** [comment]
Merge it if you think it is good.

**Pull request merged**